### PR TITLE
JTD-004 created sales employee table

### PR DESCRIPTION
**Pull request by dylanmckk**

### CODE REVIEW

**Review by dana-kainos — CHANGES_REQUESTED** [review]
'i'm not a uk employee, i don't have a national insurance number'

salesEmployeeID doesn't follow naming convention as agreed by dev team (camel case)

File: src/main/resources/db/migration/V3__create_salesemp_table.sql [hunk 1]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE SalesEmployee (
+    salesEmployeeID int AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    firstName nvarchar(50) NOT NULL,
+    lastName nvarchar(50) NOT NULL,
+    salary decimal NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V3__create_salesemp_table.sql]: Define the size of this decimal

File: src/main/resources/db/migration/V3__create_salesemp_table.sql [hunk 2]
```diff
@@ -0,0 +1,11 @@
+CREATE TABLE salesEmployees (
+    salesEmployeeId int AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    firstName nvarchar(50) NOT NULL,
+    lastName nvarchar(50) NOT NULL,
+    salary decimal NOT NULL,
+    currency char(3) NOT NULL,
+    bankAccountNo varchar(34) NOT NULL,
+    nationalInsuranceNo char(9),
+    commissionRate decimal(3,2) NOT NULL,
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V3__create_salesemp_table.sql]: is commission rate always linked to sales employee?

**Review by lukemKainos — CHANGES_REQUESTED** [review]

**Review by lukemKainos — APPROVED** [review]

**Review by dana-kainos — CHANGES_REQUESTED** [review]
remove trailing comma

**Review by dana-kainos — COMMENTED** [review]
in future ensure commit name matches format defined by team

**Review by dana-kainos — APPROVED** [review]

File: src/main/resources/db/migration/V3__create_salesemp_table.sql [hunk 3]
```diff
@@ -0,0 +1,9 @@
+CREATE TABLE salesEmployees (
+    salesEmployeeId int AUTO_INCREMENT NOT NULL PRIMARY KEY,
+    firstName varchar(50) NOT NULL,
+    lastName varchar(50) NOT NULL,
+    salary decimal(10,2) NOT NULL,
+    currency char(3) NOT NULL,
+    bankAccountNo varchar(34) NOT NULL,
+    nationalInsuranceNo char(9)
```
Comment (lukemKainos) [inline src/main/resources/db/migration/V3__create_salesemp_table.sql:8]: blank nationalInsuranceNumber? Seems like a security risk

Comment (dana-kainos) [reply src/main/resources/db/migration/V3__create_salesemp_table.sql:8]: "If you find a job before having your NIN, you can be hired as long as you are qualified to work in the UK without a working permit, that is i.e, if you have a European passport."

https://www.smallworldfs.com/en/blog/how-to-work-without-a-nin-in-the-uk

**Pull request merged**